packages/react/src/components/Tab/Tab.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import {withDesign} from '../../../.storybook/utils.ts';
+import Tab from './Tab.tsx';
+
+export const meta = {
+  component: Tab,
+  title: StoryConfig.Tab.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => <Tab {...args} />;
+
+# Tab
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+Tab component can be used with Tabs component to implement navigation between different views or sections.
+
+<Canvas>
+  <Story name="Overview" args={{label: "Sample Tab"}}>
```

Comment:
Are we satisfying all the accessibility checks? Do we need to have `aria-controls`?  
Refer - https://mui.com/material-ui/react-tabs/#accessibility

Reply:
If the `aria-controls` prop is added to `Tab` component, there should be a component with that prop value as `id` in the DOM tree. Otherwise it triggers an accessibility error. Since this story is for an individual `Tab` component, accessibility related attributes will be mentioned in `Tabs` component story.